README.md

```diff
@@ -42,9 +43,9 @@ Suggestion:
 https:// commonprofiles.care/< resource type >/<profile/extension/value set name>/< version >
```

Comment:
We should probably remove version here as well

Reply:
Fixed